docs/index.md

```diff
@@ -0,0 +1,85 @@
+
+We can use the FVM to tokenize data sets to represent their value to humanity and kickstart the data set economy. We can then use it to incentivize further value creation through data processing.
+
+With the FVM, it will be possible to write smart contracts to harvest and exchange tokens between peers to request computation, validation, transformations, feature detection, extraction, machine learning, and more.
+
+If you chain these processes together, you can compound the value of the datasets iteratively. Imagine deploying self-learning entities on-chain to steward, maintain, and curate those datasets. Data DAOs could be automatically funded by using these datasets.
+
+### Replication workers
+
+We can write smart contracts for the FVM to automatically replicate data across providers in the Filecoin network without client involvement, all while obeying a user-defined policy with specifications like region, latency, price, or other characteristics.
+
+If clients today want to replicate a piece of data with N providers, they have to perform the data transfer of that data to each provider. With smart contracts on the FVM, incentivized replication actors would offload this data transfer to clients in a trustless way.
+
+### L2 commitments
+
+In the Filecoin network:
+
+- Layer 0 is our storage and retrieval layer
+- Layer 1 is our on-chain, computation over state layer
+- Layer 2 is our off-chain, computation over data layer
+
+Currently, solutions like Textile's Filecoin bridges to NEAR, Polygon, and Ethereum are operating without a Layer 1 commitment on the Filecoin network.
+
+With smart contracts on the FVM, you can harvest Layer 2 systems to commit onto Layer 1 of the Filecoin blockchain. You can provide consensus-backed commitments to make solutions, like bridges, oracles, payment networks, rollups, and more, provable and traceable end-to-end.
+
+## Smart storage markets
+
+Smart contracts on the FVM can bring richer deal-making functionality by introducing more automation at the actor layer (Layer 1 for state change computations) and alleviating infrastructure overhead.
+
+Some possibilities are:
+
+- Auto-renewing deals: Without repeating data transfers, when deals expire or get slashed.
+- Self-repairing deals: If a provider has failed sectors and wants to recover the data, instead of having clients resend the data, replication workers can resend it automatically.
+- Retrying deals: Where a client dispatches their deal data to a depot and, for a small fee, has the deal data transmitted and satisfactorily delivered to the provider.
+
+### Time-locked retrieval
+
+The FVM enables storage deals with data that cannot be retrieved by anyone until a specific window of time elapses.
+
+### Trustless reputation systems
+
+There are so many providers in the Filecoin network, clients need help picking the providers that they want to deal with based on quality of service, guarantees, performance, latency, or region.
```

Comment:
```suggestion
There are lots of storage providers on the Filecoin network. Clients need help picking the providers that they want to deal with based on quality of service, guarantees, performance, latency, or region.
```

Reply:
@johnnymatthews Working on all these now. I know how important FVM is and I want to get it right. Thanks for adjusting the level of detail for this audience.